ALUM/ALUM/Views/SignUpPage/SignUpMenteeInfo.swift

```diff
@@ -0,0 +1,161 @@
+                        .lineSpacing(4.0)
+                        .font(.custom("Metropolis-Regular", size: 17))
+                        .foregroundColor(Color("ALUM Dark Blue"))
+
+                    Spacer()
+                }
+                .padding(.leading, 16)
+                .padding(.bottom, 2)
+                
+                ZStack {
+                    TextField("", text: $viewModel.mentee.mentorshipGoal)
+                        .padding(.init(top: 0.0, leading: 16.0, bottom: 0.0, trailing: 16.0))
+                        .frame(height: 48.0)
+                        .background(
+                            Color("ALUM White")
+                                .cornerRadius(8.0)
+                        )
+                        .overlay(
+                            RoundedRectangle(cornerRadius: 8.0).stroke(Color("NeutralGray3"), lineWidth: 1.0)
+                        )
+                }
+                .padding(.init(top: 0.0, leading: 16.0, bottom: 67.0, trailing: 16.0))
+                
+                HStack {
+                    Button("Back") {
+                        dismiss()
+                    }
+                    .buttonStyle(FilledInButtonStyle(disabled: false))
+                    .padding(.trailing, 16)
+                    .padding(.bottom, 32)
+                    .frame(width: UIScreen.main.bounds.width * 0.3)
+
+                    Button("Continue") {
+                        
+                    }
+                    .buttonStyle(OutlinedButtonStyle(disabled: false))
+                    .padding(.bottom, 32)
+                    .frame(width: UIScreen.main.bounds.width * 0.6)
+                }
+                .padding(.leading, 16)
+                .padding(.trailing, 16)
+                
+            }
+        }
+        .navigationBarItems(leading: NavigationLink(
+            destination: LoginPageView(),
+                label: {
+                    HStack {
+                        Image(systemName: "chevron.left")
+                        Text("Login")
+                            .font(.custom("Metropolis-Regular", size: 13))
+                    }
+                }
+            )
+        )
+        .navigationBarTitle(Text("Sign-Up").font(.custom("Metropolis-Regular", size: 17)), displayMode: .inline)
+        .navigationBarBackButtonHidden()
+        .onAppear {
+            viewModel.setUpMentee()
+            if viewModel.mentee.grade == "9" {
+                selectedGrade = .nine
+            } else if viewModel.mentee.grade == "10" {
+                selectedGrade = .ten
+            } else if viewModel.mentee.grade == "11" {
+                selectedGrade = .eleven
+            } else if viewModel.mentee.grade == "12" {
+                selectedGrade = .twelve
+            }
+        }
+    }
+}
+
+/*
+struct SignUpMenteeInfo_Previews: PreviewProvider {
+    static var previews: some View {
+        SignUpMenteeInfo()
+    }
+}
+*/
```

Comment:
Let's add back this preview. You should be able to initialize an empty ViewModel using SignUpViewModel()